website/source/api/secret/transit/index.html.md

```diff
@@ -806,6 +870,28 @@ supports signing.
 
 - `input` `(string: <required>)` – Specifies the **base64 encoded** input data.
```

Comment:
This is no longer a required parameter.

Reply:
I have changed "<required>" to nil - and added a sentence about either input or batch_input being needed. I'm not sure about "nil" being correct .